pandas/core/groupby/ops.py

```diff
@@ -408,6 +408,7 @@ def _masked_ea_wrap_cython_operation(
 
         # Copy to ensure input and result masks don't end up shared
         mask = values._mask.copy()
```

Comment:
can we change this to mask_in to make the naming consistent.

Reply:
how about mask_out becomes result_mask